src/classes/Bonus.ts

```diff
@@ -3,7 +3,7 @@ import dict from '@/assets/bonus_dictionary.json'
 
 interface IBonusData {
   id: string
-  val: string | number
+  val: string | number | string[]
```

Comment:
Did you run into an actual issue with this, or did you just notice that this did not match the implementation inside the Bonus class?

Reply:
Both; it didn't match the Bonus class implementation, and as a result I couldn't serialize the `Bonus.Value` field into an `IBonusData` object.